src/EditorFeatures/Core/AutomaticCompletion/AutomaticLineEnderOptionsStorage.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.AutomaticCompletion
+{
+    internal static class AutomaticLineEnderOptionsStorage
+    {
+        public static readonly Option2<bool> AutomaticLineEnder = new("dotnet_automatic_line_ender", defaultValue: true);
```

Comment:
>dotnet_automatic_line_ender [](http://example.com/codeflow?start=71&length=27)

Feels like this is missing a verb. Adding `enable` would make it consistent with `dotnet_enable_semantic_colorizer`:

`dotnet_enable_automatic_line_ender`